src/sql/workbench/contrib/assessment/browser/asmtActions.ts

```diff
@@ -294,7 +295,7 @@ export class AsmtGenerateHTMLReportAction extends Action {
 	}
 }
 
-function generateReportFileName(resultDate): string {
+function generateReportFileName(resultDate: Date): string {
 	const datetime = `${resultDate.toISOString().replace(/-/g, '').replace('T', '').replace(/:/g, '').split('.')[0]}`;
```

Comment:
Wouldn't it make sense to use a value based on the user's locale?

Reply:
This is unrelated to this PR, just wanted to give some feedback on it.

Reply:
ISO DateTimeString representation looks more generic. This is just for the default unique report name and user can change it in the save file dialog anyway.